### PR TITLE
#1623 Add SVarC OM test cases as Examples

### PR DESCRIPTION
**Pull request by MarianneSaugierRTE**
- New SVarC test cases in OM>Examples
- Update SVarC Icon so that is doesn't appear to small (tell me what you think)
- Update latex description file to add a reference to the OM test cases

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/CMakeLists.txt:1]: 2021

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/CMakeLists.txt:9]: suite of simulation tools

File: dynawo/sources/Models/Modelica/Dynawo/AdditionalIcons/package.mo [hunk 3]
```diff
@@ -55,7 +55,8 @@ package AdditionalIcons "Icons for the Dynawo library"
...
+    Icon(graphics = {Text(origin = {-1, -120}, lineColor = {0, 0, 255}, extent = {{-81, 10}, {81, -10}}, textString = "%name"), Rectangle(origin = {0, 18}, fillPattern = FillPattern.Solid, extent = {{-40, 2}, {40, -2}}), Rectangle(origin = {-20, -18}, fillPattern = FillPattern.Solid, extent = {{-20, 2}, {60, -2}}), Line(origin = {0, -21}, points = {{0, 81}, {0, -59}}), Line(origin = {-0.0916367, -80.3386}, points = {{-40, 0}, {40, 0}}), Line(origin = {-2.83665, -2.96415}, points = {{-44, -44}, {52, 52}}, thickness = 1), Line(origin = {39.2032, 49.0758}, points = {{-10, 0}, {10, 0}}, thickness = 1), Line(origin = {49.6216, 39.4941}, points = {{0, -10}, {0, 10}}, thickness = 1)}, coordinateSystem(initialScale = 0.1)));
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/AdditionalIcons/package.mo:58]: Ok for me !

**Review by cossartque — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCStepUref.mo [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+model SVarCStepUref
+  import Modelica;
+  import Dynawo;
+
+ Dynawo.Electrical.Buses.InfiniteBus infiniteBus(UPhase = 0, UPu = 1) annotation(
+    Placement(visible = true, transformation(origin = {102, 20}, extent = {{-18, -18}, {18, 18}}, rotation = -90)));
+ Dynawo.Electrical.Lines.Line line(BPu = 0, GPu = 0, RPu = 0, XPu = 0.027654) annotation(
+    Placement(visible = true, transformation(origin = {46, 20}, extent = {{-26, -26}, {26, 26}}, rotation = 0)));
+  Dynawo.Electrical.StaticVarCompensators.SVarCStandard sVarCStandard(B0Pu = 0, BMaxPu = 1.0678, BMinPu = -1.0466, BShuntPu = 0, G0Pu = 0, IMaxPu = 1, IMinPu = -1, KCurrentLimiter = 8, Kp = 1.75, Lambda = 0.01, Mode0 = Dynawo.Electrical.StaticVarCompensators.BaseControls.Mode.RUNNING_V, P0Pu = 0, Q0Pu = 0, SNom = 250, Ti = 0.003428, U0Pu = 1, UBlock = 5, UNom = 225, URef0 = 225, URefDown = 225, URefUp = 225, UThresholdDown = 218, UThresholdUp = 240, UUnblockDown = 180, UUnblockUp = 270, i0Pu = Complex(0, 0), s0Pu = Complex(0, 0), tThresholdDown = 0, tThresholdUp = 60, u0Pu = Complex(1, 0)) annotation(
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCStepUref.mo:23]: In the dynawo nrt URefUp = 230 and URefDown = 220

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCLoadVarQ.mo [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+model SVarCLoadVarQ
+  import Modelica;
+  import Dynawo;
+
+  Dynawo.Electrical.Buses.InfiniteBus infiniteBus(UPhase = 0, UPu = 1) annotation(
+    Placement(visible = true, transformation(origin = {102, 20}, extent = {{-18, -18}, {18, 18}}, rotation = -90)));
+  Dynawo.Electrical.Lines.Line line(BPu = 0, GPu = 0, RPu = 0, XPu = 0.027654) annotation(
+    Placement(visible = true, transformation(origin = {46, 20}, extent = {{-26, -26}, {26, 26}}, rotation = 0)));
+  Dynawo.Electrical.StaticVarCompensators.SVarCStandard sVarCStandard(B0Pu = 0, BMaxPu = 1.0678, BMinPu = -1.0466, BShuntPu = 0, G0Pu = 0, IMaxPu = 1, IMinPu = -1, KCurrentLimiter = 8, Kp = 1.75, Lambda = 0.01, Mode0 = Dynawo.Electrical.StaticVarCompensators.BaseControls.Mode.STANDBY, P0Pu = 0, Q0Pu = 0, SNom = 250, Ti = 0.003428, U0Pu = 1, UBlock = 5, UNom = 225, URef0 = 225, URefDown = 220, URefUp = 225, UThresholdDown = 218, UThresholdUp = 240, UUnblockDown = 180, UUnblockUp = 270, i0Pu = Complex(0, 0), s0Pu = Complex(0, 0), tThresholdDown = 0, tThresholdUp = 60, u0Pu = Complex(1, 0)) annotation(
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCLoadVarQ.mo:23]: In dynawo nrt, URefUp = 230

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCLoadVarQLarge.mo [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+model SVarCLoadVarQLarge
+  import Modelica;
+  import Dynawo;
+
+  Dynawo.Electrical.Buses.InfiniteBus infiniteBus(UPhase = 0, UPu = 1) annotation(
+    Placement(visible = true, transformation(origin = {102, 20}, extent = {{-18, -18}, {18, 18}}, rotation = -90)));
+  Dynawo.Electrical.Lines.Line line(BPu = 0, GPu = 0, RPu = 0, XPu = 0.027654) annotation(
+    Placement(visible = true, transformation(origin = {46, 20}, extent = {{-26, -26}, {26, 26}}, rotation = 0)));
+  Dynawo.Electrical.StaticVarCompensators.SVarCStandard sVarCStandard(B0Pu = 0, BMaxPu = 1.0678, BMinPu = -1.0466, BShuntPu = 0, G0Pu = 0, IMaxPu = 1, IMinPu = -1, KCurrentLimiter = 8, Kp = 1.75, Lambda = 0.01, Mode0 = Dynawo.Electrical.StaticVarCompensators.BaseControls.Mode.STANDBY, P0Pu = 0, Q0Pu = 0, SNom = 250, Ti = 0.003428, U0Pu = 1, UBlock = 5, UNom = 225, URef0 = 225, URefDown = 225, URefUp = 225, UThresholdDown = 218, UThresholdUp = 240, UUnblockDown = 180, UUnblockUp = 270, i0Pu = Complex(0, 0), s0Pu = Complex(0, 0), tThresholdDown = 0, tThresholdUp = 60, u0Pu = Complex(1, 0)) annotation(
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCLoadVarQLarge.mo:23]: URefUp = 230 in dynawo nrt

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCModeChange.mo [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+model SVarCModeChange
+  import Modelica;
+  import Dynawo;
+
+  Dynawo.Electrical.Buses.InfiniteBus infiniteBus(UPhase = 0, UPu = 1) annotation(
+    Placement(visible = true, transformation(origin = {102, 20}, extent = {{-18, -18}, {18, 18}}, rotation = -90)));
+  Dynawo.Electrical.Lines.Line line(BPu = 0, GPu = 0, RPu = 0, XPu = 0.027654) annotation(
+    Placement(visible = true, transformation(origin = {46, 20}, extent = {{-26, -26}, {26, 26}}, rotation = 0)));
+  Dynawo.Electrical.StaticVarCompensators.SVarCStandard sVarCStandard(B0Pu = 0, BMaxPu = 1.0678, BMinPu = -1.0466, BShuntPu = 0, G0Pu = 0, IMaxPu = 1, IMinPu = -1, KCurrentLimiter = 8, Kp = 1.75, Lambda = 0.01, Mode0 = Dynawo.Electrical.StaticVarCompensators.BaseControls.Mode.RUNNING_V, P0Pu = 0, Q0Pu = 0, SNom = 250, Ti = 0.003428, U0Pu = 1, UBlock = 5, UNom = 225, URef0 = 225, URefDown = 225, URefUp = 225, UThresholdDown = 218, UThresholdUp = 240, UUnblockDown = 180, UUnblockUp = 270, i0Pu = Complex(0, 0), s0Pu = Complex(0, 0), tThresholdDown = 0, tThresholdUp = 60, u0Pu = Complex(1, 0)) annotation(
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCModeChange.mo:23]: URefUp = 230 and URefDown = 220 in dynawo nrt

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCFaultImp.mo [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+model SVarCFaultImp
+  import Modelica;
+  import Dynawo;
+
+  Dynawo.Electrical.Buses.InfiniteBus infiniteBus(UPhase = 0, UPu = 1) annotation(
+    Placement(visible = true, transformation(origin = {102, 20}, extent = {{-18, -18}, {18, 18}}, rotation = -90)));
+  Dynawo.Electrical.Lines.Line line(BPu = 0, GPu = 0, RPu = 0, XPu = 0.027654) annotation(
+    Placement(visible = true, transformation(origin = {46, 20}, extent = {{-26, -26}, {26, 26}}, rotation = 0)));
+  Dynawo.Electrical.StaticVarCompensators.SVarCStandard sVarCStandard(B0Pu = 0, BMaxPu = 1.0678, BMinPu = -1.0466, BShuntPu = 0, G0Pu = 0, IMaxPu = 1, IMinPu = -1, KCurrentLimiter = 8, Kp = 1.75, Lambda = 0.01, Mode0 = Dynawo.Electrical.StaticVarCompensators.BaseControls.Mode.RUNNING_V, P0Pu = 0, Q0Pu = 0, SNom = 250, Ti = 0.003428, U0Pu = 1, UBlock = 5, UNom = 225, URef0 = 225, URefDown = 225, URefUp = 225, UThresholdDown = 218, UThresholdUp = 240, UUnblockDown = 180, UUnblockUp = 270, i0Pu = Complex(0, 0), s0Pu = Complex(0, 0), tThresholdDown = 0, tThresholdUp = 60, u0Pu = Complex(1, 0)) annotation(
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCFaultImp.mo:23]: URefUp = 230 and URefDown = 220 in dynawo nrt

**Review by cossartque — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCStepUref.mo (hunk 4 above)
Comment (MarianneSaugierRTE) [reply dynawo/sources/Models/Modelica/Dynawo/Examples/SVarC/SVarCStepUref.mo]: done

**Review by cossartque — APPROVED** [review]

**Pull request merged**